Opcodes/dsputil.c

```diff
@@ -40,10 +40,11 @@
     /* where callers pass s/2+1, this recreates the parent fft frame size */
 
 /* assumes that FFTsize is an integer multiple of 4 */
-void Polar2Real_PVOC(CSOUND *csound, MYFLT *buf, int32_t FFTsize)
+void Polar2Real_PVOC(CSOUND *csound, MYFLT *buf, void *setup)
```

Comment:
Could we change this from `void* setup` to `CSOUND_FFT_SETUP *setup`?

Reply:
done